common/models/src/utils.rs

```diff
@@ -91,3 +91,7 @@ impl SeqIdGenerator {
             .fetch_add(1, std::sync::atomic::Ordering::Relaxed)
     }
 }
+
+pub fn splice_address(address: String, port: u32) -> String {
+    address + ":" + &port.to_string()
```

Comment:
To generate a string, macro format!() is better.

#### Example

```rust
format!("{}:{}", address, port)
```

Reply:
ok